spec/src/main/asciidoc/microprofile-config-spec.asciidoc

```diff
@@ -30,8 +13,11 @@ ifdef::backend-pdf[]
 :pagenums:
 endif::[]
 
+// == License
+:sectnums!:
+include::license-efsl.adoc[]
 
-include::license-alv2.asciidoc[]
```

Comment:
Shouldn't we do the same thing here and just change the license when doing a Final release?

Reply:
From my understanding, Jakarta Specs use EFSL licenses all the time. Kevin did say the generated doc/html not source files uses EFSL. The generated pdf/htmls means all releases including the local generated files. @kwsutter please can you confirm?

Reply:
@kwsutter mentioned this:

> it's just a requirement that the final release version of the Specification and Javadoc use the EFSL license. Non-final versions of the Spec and Javadoc can use Apache v2, EPL, or whatever the project is currently using for the source.

If we can use the EFSL all the time in the spec files it is fine for me.

Reply:
This is kind of confusing...  And, to be honest, you may not find consistency across all of the Jakarta EE specs... The requirement is that the Final generated Specifications and Javadocs have to contain the EFSL.  These are the Final specs and javadoc available via our Specifications page:  https://jakarta.ee/specifications/ .  When we produced our Milestone1 release for Jakarta EE 9, most of the Specifications and Javadocs also used the EFSL license, but we were not as picky with verifying that.  Most Specification projects, if you just clone the repo and rebuild the specs as a developer, you'll end up with EPL (or Apache in some cases) in the generated Specs and Javadoc.

The key requirement is that the Final released version of the Specs and Javadocs have to contain the EFSL.

Reply:
> Most Specification projects, if you just clone the repo and rebuild the specs as a developer, you'll end up with EPL (or Apache in some cases) in the generated Specs and Javadoc.

@Emily-Jiang pointed out that this is not always the case (at least for the Platform specs).  For the Platform, we always use the EFSL in the generated Spec pdf and html.  But, for the API, we will use the EPL for normal development builds of the Javadoc and use the EFSL when producing the Final version of Javadoc.

Reply:
ok, thank you @kwsutter ! I did clone a few Jakarta-EE specs and realised that the specs always use EFSL. @kwsutter confirmed that he was a bit confused and confirmed that spec pdf/html always use EFSL and javadoc is the one that need to be differentiated.

Reply:
Based on the previous comments, no new actions are to be performed.